### PR TITLE
#6536 - Abattoir Blues

### PR DESCRIPTION
**Pull request by Miaplacidus**
Replace the old cow-based loading spinner with our new dots-based spinner

## Link to Issue
Closes: #6536

## Description of Changes
- replaces instances of old spinner component with new spinner component

## Test Plan
- visit pages where new spinner is used. Confirm that they look as expected

### CODE REVIEW

File: packages/commonwealth/client/scripts/App.tsx [hunk 1]
```diff
@@ -9,13 +9,13 @@ import { RouterProvider } from 'react-router-dom';
 import { ToastContainer } from 'react-toastify';
 import { queryClient } from 'state/api/config';
 import { featureFlags } from './helpers/feature-flags';
-import { CWIcon } from './views/components/component_kit/cw_icons/cw_icon';
+import CWLoadingSpinner from './views/components/component_kit/new_designs/CWLoadingSpinner';
```
Comment (masvelio) [inline packages/commonwealth/client/scripts/App.tsx:12]: some time ago we had a conversation with design team about spinners nomenclature and there was a suggestion from design team to change the name of our components:
- CWLoadingSpinner => CWCircleMultiplySpinner
- CWCircleSpinner => CWCircleRingSpinner

Do you mind changing this in this PR. Or creating another ticket for this? [See context](https://commonxyz.slack.com/archives/C052DMYFKDL/p1706722952215969?thread_ts=1706721213.866819&cid=C052DMYFKDL)

**Review by masvelio — APPROVED** [review]
small comments but works fine in general!

File: packages/commonwealth/client/scripts/views/components/component_kit/cw_spinner.tsx [hunk 2]
```diff
@@ -1,21 +0,0 @@
-import React from 'react';
-
-import 'components/component_kit/cw_spinner.scss';
```
[the remark this answers is not in the document]
Comment (Miaplacidus) [reply packages/commonwealth/client/scripts/views/components/component_kit/cw_spinner.tsx:3]: Thanks, @masvelio!
Updated!

File: packages/commonwealth/client/scripts/App.tsx [hunk 3]
```diff
@@ -9,13 +9,13 @@ import { RouterProvider } from 'react-router-dom';
...
 
 const Splash = () => {
   return (
     <div className="Splash">
       {/* This can be a moving bobber, atm it is still */}
-      <CWIcon iconName="cow" iconSize="xxl" />
+      <CWLoadingSpinner center />
```
[the remark this answers is not in the document]
Comment (Miaplacidus) [reply packages/commonwealth/client/scripts/App.tsx]: Done!

File: packages/commonwealth/client/scripts/App.tsx (hunk 1 above)
Comment (Miaplacidus) [reply packages/commonwealth/client/scripts/App.tsx:12]: @masvelio 
Created a card around those two changes and tagged you in them.

**Review by Israellund — APPROVED** [review]
LGTM

**Pull request merged**